components/steps/Profile.tsx

```diff
@@ -208,7 +208,7 @@ const ProfileStep: React.FunctionComponent<Props> = props => {
               defaultValue={profile.school || getSchoolFromEmail(profile.email)}
               required
               ref={schoolRef}
-              disabled={submitted}
+              disabled={submitted || !!getSchoolFromEmail(profile.email)}
```

Comment:
added this to prevent users from editing their school if we can auto-detect it from their e-mail